gnomad_qc/v3/create_release/get_het_nonref_variants_to_fix.py

```diff
@@ -0,0 +1,181 @@
+)
+logger = logging.getLogger("get_impacted_variants")
+logger.setLevel(logging.INFO)
+
+
+def get_het_non_ref_impacted_var(
+    mt: hl.MatrixTable, info_ht: hl.Table, freq_ht: hl.Table
+) -> None:
+    """
+    Filter to variants where homalt hotfix incorrectly adjusts het nonref genotype calls.
+
+    :param hl.MatrixTable mt: Raw, split MatrixTable annotated with original genotype het nonref status.
+    :param hl.Table info_ht: Info Hail Table containing AS_lowqual information.
+    :param hl.Table freq_ht: Hail Table containing gnomAD v3.0 frequency information.
+    :return: None
+    """
+    logger.info("Filtering to common (AF > 0.01) variants...")
+    mt = mt.annotate_rows(AF=freq_ht[mt.row_key].freq[0].AF)
+    mt = mt.filter_rows(mt.AF > 0.01)
+
+    logger.info(
+        "Filtering to variants with at least one het nonref call and checkpointing..."
+    )
+    mt = mt.filter_rows(hl.agg.sum(mt._het_non_ref) >= 1)
+
+    logger.info(
+        "Adding new genotype entries with original homalt hotfix and het nonref-aware hotfix..."
+    )
+    mt = mt.annotate_entries(
+        GT_hotfix=hl.if_else(
+            mt.GT.is_het() & (mt.AF > 0.01) & (mt.AD[1] / mt.DP > 0.9),
+            hl.call(1, 1),
+            mt.GT,
+        ),
+        GT_hetnonref_fix=hl.if_else(
+            mt.GT.is_het()
+            & ~mt._het_non_ref
+            & (mt.AF > 0.01)
+            & (mt.AD[1] / mt.DP > 0.9),
+            hl.call(1, 1),
+            mt.GT,
+        ),
+    )
+    logger.info("Checking homozygote counts using the different genotypes...")
+    # Taking index 1 of gt_stats because it contains an entry for the reference allele at index 0
+    # homozygote_count (tarray of tint32) -
+    # Homozygote genotype counts for each allele, including the reference. Only diploid genotype calls are counted.
+    # (from hail docs, https://hail.is/docs/0.2/aggregators.html#hail.expr.aggregators.call_stats)
+
+    mt = mt.annotate_rows(gt_stats=hl.agg.call_stats(mt.GT_hotfix, mt.alleles))
+    mt = mt.transmute_rows(hom_count_hotfix=mt.gt_stats.homozygote_count[1])
+
+    mt = mt.annotate_rows(gt_stats=hl.agg.call_stats(mt.GT_hetnonref_fix, mt.alleles))
+    mt = mt.transmute_rows(hom_count_hetnonref_fix=mt.gt_stats.homozygote_count[1])
+
+    logger.info(
+        "Filtering to rows where homalt hotfix erroneously adjusts het nonref calls..."
+    )
+    mt = mt.filter_rows(mt.hom_count_hotfix != mt.hom_count_hetnonref_fix)
+
+    return mt.rows()
+
+
+def main(args):
+    """
+    Script used to get variants impacted by homalt hotfix.
+    """
+
+    hl.init(log="/get_impacted_variants.log", default_reference="GRCh38")
+
+    logger.info("Reading sparse MT and metadata table with release only samples...")
+    mt = get_gnomad_v3_mt(
+        key_by_locus_and_alleles=True, samples_meta=True, release_only=True,
+    ).select_entries(*SPARSE_ENTRIES)
+
+    if args.test:
+        logger.info("Filtering to two partitions on chr20")
```

Comment:
```suggestion
        logger.info("Filtering to two partitions on chr20...")
```
(not important, just for consistency with your other loggers)

Reply:
I will also change this to chr1 as I did for the other